### ISSUE TITLE
npm test should work

### ISSUE THREAD

**Issue by nponeccop** [streamcode9/node-discrete-spsa#9]

**Comment by streamcode9** [comment]
```
/home/travis/build/streamcode9/node-discrete-spsa/node_modules/mocha/lib/utils.js:628

        throw new Error("cannot resolve path (or pattern) '" + path + "'");

              ^

Error: cannot resolve path (or pattern) 'test'

    at Object.lookupFiles (/home/travis/build/streamcode9/node-discrete-spsa/node_modules/mocha/lib/utils.js:628:15)

    at runAgain (/home/travis/build/streamcode9/node-discrete-spsa/node_modules/mocha/bin/_mocha:326:30)

    at Array.forEach (native)

    at Object.<anonymous> (/home/travis/build/streamcode9/node-discrete-spsa/node_modules/mocha/bin/_mocha:325:6)

    at Module._compile (module.js:456:26)

    at Object.Module._extensions..js (module.js:474:10)

    at Module.load (module.js:356:32)

    at Function.Module._load (module.js:312:12)

    at Function.Module.runMain (module.js:497:10)

    at startup (node.js:119:16)

    at node.js:945:3
```

**Comment by nponeccop** [comment]
http://stackoverflow.com/a/20384739/805266 is one solution to try

**Comment by nponeccop** [comment]
Fixed in 921e156ee960ed58e5215c662437a75899085c48
